src/main/java/org/toradocu/translator/Matcher.java

```diff
@@ -168,7 +168,8 @@ String predicateMatch(
       }
     }
 
-    if (match.equals("target==null")) { // Condition "target==null" is indeed not correct.
+    if (match.equals(
+        "receiverObjectID==null")) { // Condition "receiverObjectID==null" is indeed not correct.
```

Comment:
Shouldn’t be `Configuration.RECEIVER==null`?

Reply:
Right, it was an oversight